errors.dd

```diff
@@ -173,6 +173,10 @@ $(DL
 Using exceptions to handle errors leads to another issue - how to write
 exception safe programs. $(DPLLINK exception-safe.html, Here's how).
 
+D uses $(DPLLINK statement.html#TryStatement, try, catch, finally statements)
+to handle exceptions. You can raise Exceptions using throw, as in
+throw new Exception("oops!");
```

Comment:
A basic try/catch/throw paragraph should be at the beginning not at the bottom of this text.